### DISCUSSION TITLE
Error Message on version of .NET Framework - SteamOS

### DISCUSSION THREAD

**Discussion by mygitH1** [DeckCheatz/wemod-launcher#109] [Q&A] [answered]
Hello, 

I am getting this error messge stating "You need to have the correct version of the .net framework installed". 

Note I have utilized ProtonQT and wine tricks. I believe I have installed it correctly but for some reason its not picking up the .net framework I have installed through winetricks. 

Maybe the application is not locating it and its as simple as mapping it or adding somewhere within its directory. 

I would appreciate some help or advice here. 

![1000044733](https://github.com/user-attachments/assets/d4178402-2f30-44b0-b3b1-116acad7127d)

**Comment by marvin1099** [comment, marked as answer]
You will need to delete the game prefix and then retry

**Comment by marvin1099** [comment]
For some additional info,
When the game prefix is build it installs a few things, 
so delete the prefix, 
then click download when asked,
After that a working prefix will be downloaded.

Hope that solved the issue,
closing this discussion now.
